tests/cli/test_cli.py

```diff
@@ -1357,6 +1357,21 @@ def test_run(ctx):
         2 + 2 + 1, _run_command([cl, 'cat', remote_uuid])
     )  # 2 header lines, 1 stdout file, 1 stderr file, 1 item at bundle target root
 
+    # test metadata
+    uuid = _run_command(
+        [
+            cl,
+            "run",
+            "'for i in {1..60}; do sleep 1; done'",
+            "--request-memory",
+            "15m",
+            "--request-docker-image",
+            "python:3.6.10-buster",
+        ]
+    )
+    check_not_equals(get_info(uuid, 'cpu_usage'), '0.0')
+    check_not_equals(get_info(uuid, 'memory_limit'), '0')
```

Comment:
Can we instead check to ensure that when we parse it as an int, it's greater than 0?

`assert int(get_info(uuid, 'memory_limit')) > 0`

This way, we'll ensure the returned value is correct (as opposed to something else like `None`)